controllers/user.js

```diff
@@ -689,6 +689,49 @@ async function getRemoveMoreTimePage(req, res) {
   });
 }
 
+function alphabetizeByBroker(data) {
+  return data.sort(function (a, b) {
+    if (a.broker < b.broker) {
+      return -1;
+    }
+    if (a.broker > b.broker) {
+      return 1;
+    }
+    return 0;
+  });
+}
+
+function sortRemovalData(removalData) {
+  const completeItems = removalData.filter((removalItem) => {
+    return removalItem.status === REMOVAL_STATUS["COMPLETE"].id;
+  });
+
+  alphabetizeByBroker(completeItems);
+
+  const blockedItems = removalData.filter((removalItem) => {
+    return (
+      removalItem.current_step === JS_CONSTANTS.REMOVAL_STEP["BLOCKED"].code
+    );
+  });
+
+  alphabetizeByBroker(blockedItems);
+
+  const activeItems = removalData.filter((removalItem) => {
+    return (
+      removalItem.status !== REMOVAL_STATUS["COMPLETE"].id &&
+      removalItem.current_step !== JS_CONSTANTS.REMOVAL_STEP["BLOCKED"].code
```

Comment:
Given that we do this in a couple places, I wonder if it's ripe for some helper functions:

```js
function isComplete(status) {
  return status === REMOVAL_STATUS.COMPLETE.id;
}

function isBlocked(step) {
  return step === JS_CONSTANTS.REMOVAL_STEP.BLOCKED.code;
}

...

return (
  !isComplete(removalItem.status) &&
  !isBlocked(removalItem.current_step)
);
```